sdk/face/ai-vision-face-rest/samples-dev/livenessWithVerifyWithImage.ts

```diff
@@ -0,0 +1,113 @@
+import { randomUUID } from 'crypto';
+
+import { AzureKeyCredential } from '@azure/core-auth';
+
+import createFaceClient, {
+    CreateLivenessWithVerifySession200Response,
+    DeleteLivenessWithVerifySession200Response,
+    GetLivenessWithVerifySessionAuditEntries200Response,
+    GetLivenessWithVerifySessionResult200Response,
+    CreateLivenessWithVerifySessionContentParametersPartDescriptor,
+    CreateLivenessWithVerifySessionContentVerifyImagePartDescriptor,
+    GetLivenessWithVerifySessions200Response,
+} from '@azure-rest/ai-vision-face';
+import { readFileSync } from 'fs';
+
+/**
+ * This sample demonstrates how to create a liveness detection session.
+ *
+ * @summary creates a liveness detection session
+ */
+
+const pressAnyKeyToContinue = () => new Promise<void>(resolve => {
+    const { stdin } = process;
+    const { isRaw } = stdin;
+    stdin.setRawMode(true);
+    stdin.resume();
+    stdin.once('data', () => {
+        stdin.pause();
+        stdin.setRawMode(isRaw);
+        resolve();
+    });
+});
+
+const waitForLivenessRequest = async () => {
+    // Wait for request from client device.
+};
+
+const sendTokenToClientDevices = async (token: string) => {
+    // Send the token to client devices.
+};
+
+const waitForLivenessSessionComplete = async () => {
+    console.log('Please refer to https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness and use the mobile client SDK to perform liveness detection on your mobile application.');
+    console.log('Press any key to continue when you complete these steps to run sample to get session results...');
+    await pressAnyKeyToContinue();
+};
+
+async function main() {
+    // This sample follows the documentation: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness
+    // We will follow the steps in https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness#orchestrate-the-liveness-solution to demo the sample code in app server.
+
+    const endpoint = process.env['FACE_ENDPOINT'] ?? '<endpoint>';
+    const apikey = process.env['FACE_APIKEY'] ?? '<apikey>';
+    const credential = new AzureKeyCredential(apikey);
+    const client = createFaceClient(endpoint, credential);
+
+    // 1. A client device will send a request to start liveness check to us.
+    await waitForLivenessRequest();
+
+    // 2.Send a request to Face API to create a liveness detection session.
+    const createLivenessSessionResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions').post({
+        contentType: 'multipart/form-data',
+        body: [
+            new CreateLivenessWithVerifySessionContentVerifyImagePartDescriptor({
+                name: 'VerifyImage',
+                body: readFileSync('samples-dev/data/detection1.jpg'),
+            }),
+            new CreateLivenessWithVerifySessionContentParametersPartDescriptor({
```

Comment:
if we remove the class wrapper, we don't need to create the payload this way ?

Reply:
But we won't be able to give `filename` a default value.